### PR TITLE
serverless to 4-7

### PR DESCRIPTION
**Pull request by ryanj**
* Updating the "Intro to Serverless" scenario to openshift-4-7
* Including a fix for #995, #987

### CODE REVIEW

File: developing-on-openshift/serverless/01-prepare.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [serverless-install-script]: https://github.com/openshift-labs/learn-katacoda/blob/master/developing-on-openshift/serverless/assets/01-prepare/install-serverless.bash
-[olm-docs]: https://docs.openshift.com/container-platform/4.4/operators/olm-adding-operators-to-cluster.html
+[olm-docs]: https://docs.openshift.com/container-platform/4.7/operators/olm-adding-operators-to-cluster.html
```
Comment (btannous) [inline developing-on-openshift/serverless/01-prepare.md:2]: https://docs.openshift.com/container-platform/4.7/operators/olm-adding-operators-to-cluster.html
404 not found. update this url

File: developing-on-openshift/serverless/01-prepare.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Now logged in as an admin, open the *Operator Hub* and find and select the `Open
 >
 > **Tip:** *You can find more information on how to add operators on the [OpenShift OLM Documentation Page][olm-docs].*
 
-Next, install the Serverless Operator into the `openshift-operators` project.  We choose the update channel of `4.4` to match the OpenShift Container Platform version that we are using.
+Next, install the Serverless Operator into the `openshift-operators` project.  We choose the update channel of `4.7` to match the OpenShift Container Platform version that we are using.
```
Comment (btannous) [inline developing-on-openshift/serverless/01-prepare.md:28]: This is updated now.  the update channel `4.7` would work, but `stable` is recommended by the docs now (and will be less updating later down the road).

https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/serverless/administration-guide#installing-openshift-serverless

File: developing-on-openshift/serverless/assets/01-prepare/operator-subscription.yaml [hunk 3]
```diff
@@ -1,12 +1,10 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
-apiVersion: operators.coreos.com/v1alpha1
-kind: Subscription
 metadata:
   name: serverless-operator
   namespace: openshift-operators
 spec:
-  channel: "4.4"
+  channel: "4.7"
```
Comment (btannous) [inline developing-on-openshift/serverless/assets/01-prepare/operator-subscription.yaml:7]:
```suggestion
  channel: "stable"
```

File: developing-on-openshift/serverless/01-prepare.md (hunk 2 above)
Comment (btannous) [reply developing-on-openshift/serverless/01-prepare.md:28]:
```suggestion
Next, install the Serverless Operator into the `openshift-operators` project.  We choose the update channel of `stable`.
```

File: developing-on-openshift/serverless/01-prepare.md (hunk 1 above)
Comment (btannous) [reply developing-on-openshift/serverless/01-prepare.md:2]:
```suggestion
[olm-docs]: https://docs.openshift.com/container-platform/4.7/operators/understanding/olm/olm-understanding-olm.html
```

**Review by btannous — CHANGES_REQUESTED** [review]

File: developing-on-openshift/serverless/01-prepare.md (hunk 1 above)
Comment (ryanj) [reply developing-on-openshift/serverless/01-prepare.md]: thanks! should be fixed now

File: developing-on-openshift/serverless/01-prepare.md (hunk 2 above)
Comment (ryanj) [reply developing-on-openshift/serverless/01-prepare.md]: fixed

File: developing-on-openshift/serverless/assets/01-prepare/operator-subscription.yaml (hunk 3 above)
Comment (ryanj) [reply developing-on-openshift/serverless/assets/01-prepare/operator-subscription.yaml]: got it

**Comment by cedricclyburn** [comment]
Looks good to me! Will merge and verify from user POV

**Pull request merged**